src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({onClickHandler, text, ...rest}) => <button onClick={onClickHandler} {...rest}>{text}</button>;
+
+Button.propTypes = {
+    onClickHandler: PropTypes.func.isRequired,
+}
+Button.defaultProps = {
```

Comment:
this is not needed, but we could think about {children} props combined with `text` prop (there are needed some small conditionals to check what was given to the compoent and render it properly). Why? it would give an opportunity to add for example icon ;) 
text is not verified in props.